src/core/db/backend/pg.py

```diff
@@ -221,6 +227,15 @@ def add_collaborator(self, repo, collaborator, db_privileges=[]):
                   collaborator, repo, privileges_str, collaborator]
         params = tuple(map(lambda x: AsIs(x), params))
         res = self.execute_sql(query, params)
+
+        query = ('BEGIN;'
```

Comment:
This also has one too many spaces, though that's more of a silly codeclimate issue.
https://codeclimate.com/github/datahuborg/datahub/pull/204